### ISSUE TITLE
How can i integrate Novel to my VUE project?

### ISSUE THREAD

**Issue by allenho333** [naveennaidu/novel-vue#1]
How can i integrate Novel to my VUE project?

**Comment by naveennaidu** [comment]
You can check out the documentation here: https://github.com/naveennaidu/novel-vue/tree/main/package
Also, there is a Nuxt 3 based demo project using novel-vue package: https://github.com/naveennaidu/novuel

**Comment by allenho333** [comment]
Thx i see and integrate to my VUE project

Can i use our LLM model instead of OPEN AI?

we want the AI function to connect to our own LLM model

________________________________

**Comment by naveennaidu** [comment]
Yes, You can see the `api/generate` endpoint I used https://github.com/naveennaidu/novuel/blob/main/server/api/generate.ts. you can replace openai with any other LLM here

**Comment by allenho333** [comment]
I see,

Can I add more extension so I can also add image or drag and drop function?

Or do you have any code snippet, I try to add extension attribute to the Editer component but I did not work

Can u provide some code snippet ?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by naveennaidu** [comment]
Image, drag and drop extensions need to be added. I don't have a code snippet, you can check out https://github.com/steven-tey/novel 

Do you want take it up and create a PR?

**Comment by allenho333** [comment]
Yeah Sure can

But u mean new functions such  as add photo can not be added through the property extensions?

Or can u provide me the file for me to look at?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by naveennaidu** [comment]
Great!

You can look at this [extensions file](https://github.com/naveennaidu/novel-vue/blob/main/package/src/components/extensions/index.ts) to add image functionality. Currently it's commented
